### DISCUSSION TITLE
Please ! add a pop-up dialog Corner Radius!

### DISCUSSION THREAD

**Discussion by wj8400684** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3088] [Ideas]
![image](https://user-images.githubusercontent.com/39239954/218840708-a09934b3-f7b6-4816-b16f-5efcdda72a68.png)

**Comment by nicolaihenriksen** [comment]
@wj8400684 Seems like a reasonable request. I have added a PR (#3103) introducing a property to control this.

**Comment by nicolaihenriksen** [comment]
PR merged and corner radius is now available.
